python/ray/serve/utils.py

```diff
@@ -301,9 +301,26 @@ def try_schedule_resources_on_nodes(
         for node_id, node_resource in ray_resource.items():
             # Check if we can schedule on this node
             feasible = True
-            for key, count in resource_dict.items():
-                if node_resource.get(key, 0) - count < 0:
-                    feasible = False
+
+            # Legacy memory is treated in 50 byte chunks
+            if int(ray.__version__.split(".")[0]) >= 1:
+                for key, count in resource_dict.items():
+                    # Fix legacy behaviour in memory
+                    if "memory" in key:
+                        memory_resource = node_resource.get(key, 0)
+                        if memory_resource > 0:
+                            # Convert from chunks of 50mb to bytes
+                            memory_resource = memory_resource * 5e+7
```

Comment:
can we use https://github.com/ray-project/ray/blob/master/python/ray/ray_constants.py#L69 `MEMORY_RESOURCE_UNIT_BYTES` from ray.ray_constants?